tests/test_fellow.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import nose
+
+from models.fellow import Fellow
+
+class TestFellow(unittest.TestCase):
+	"""Test cases for the Fellow class"""
+
+	def setUp(self):
+		"""Setup a new Fellow object to be used for testing"""
+		self.fellow = Fellow('Brian','Mosigisi')
+
+	def test_fellow_instance(self):
+		"""Tests if the object created is an instance of the Fellow class"""
+		self.assertIsInstance(self.fellow,Fellow)
```

Comment:
I see you are testing if `self.fellow` is an instance of `Fellow` class. Is this really necessary?